src/ResourceManager/Relay/Commands.Relay/Commands.Relay.csproj

```diff
@@ -41,10 +41,71 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Management.Relay, Version=0.0.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@v-Ajnava please revert the changes made in this file. I assume they are a result of a merge conflict after our Common.Dependencies PR went through. The only thing that should be in csproj files should be the management libraries that the project depends on. In this case, it should just be the Relay management library. Everything else is defined in [`Common.Dependencies.targets`](https://github.com/Azure/azure-powershell/blob/preview/tools/Common.Dependencies.targets)